examples/core-hello-world/src/main/scala/com/mesosphere/usi/examples/CoreHelloWorldFramework.scala

```diff
@@ -43,7 +33,7 @@ import scala.sys.SystemProperties
   */
 class CoreHelloWorldFramework(conf: Config) extends StrictLogging {
   implicit val system = ActorSystem()
-  implicit val mat = ActorMaterializer()
+  implicit val mat = ActorMaterializer(ActorMaterializerSettings(system).withDebugLogging(true))
```

Comment:
is debug logging the way to go for production use?

Reply:
good catch!